README.md

```diff
@@ -161,21 +161,21 @@ A known `mnemonic` can be used when restoring a wallet, or a new `mnemonic` can
 
 ```dart
                                                 
-final response  =  await BdkFlutter().createWallet( mnemonic:"puppy interest whip tonight dad never sudden response push zone pig patch",
-                                                    password: "password",
-                                                    network: Network.TESTNET,
-                                                    blockchainConfig: BlockchainConfig.electrum
-						    (
-						        config: ElectrumConfig(
+final aliceWallet  =  await BdkFlutter().createWallet( mnemonic:"puppy interest whip tonight dad never sudden response push zone pig patch",
```

Comment:
The spacing on most of the readme appears to be off on my local copy/IDE but not sure why.

<img width="1310" alt="spacing" src="https://user-images.githubusercontent.com/39974688/199566843-f81bb107-db4a-4565-a68f-ed3f45b6bea9.png">